docs/release_notes.rst

```diff
@@ -34,3 +22,40 @@ runs on most Windows, Linux and macOS platforms.
 `main Connector
 repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
+Version 2.0.0 (TODO Numebring)
+------------------------------
+
+What's New in 2.0.0
+~~~~~~~~~~~~~~~~~~~
+
+What's Fixed in 2.0.0
+~~~~~~~~~~~~~~~~~~~~~
+
+CON-163: Creating two instances of Connector resulted in a license error 
```

Comment:
For "What's Fixed" - Jira #s should be at the end of the description, not in the title, like this:
[RTI Issue ID CON-163]